config/dev.exs

```diff
@@ -122,6 +123,7 @@ config :phoenix, :stacktrace_depth, 20
 config :phoenix, :plug_init_mode, :runtime
 
 config :fz_http,
+  wireguard_allowed_ips: "172.28.0.0/16",
```

Comment:
By default, generate client configs that route to the Docker net. The `client` container config has `0.0.0.0/0,::/0` to test tunneling out to the broader internet, but this will allow devs to test using their Host's WireGuard client.